src/components/Nav/NavUtils.tsx

```diff
@@ -1,6 +1,8 @@
 import { GraphParamsType, NodeType } from '../../types/Graph';
 import { store } from '../../store/ConfigStore';
 
+const URLSearchParams = require('url-search-params');
```

Comment:
We use URLSearchParams already in several places but use the native object, so I would say, no need to `require` it.
(if old browser support is wanted at some point for `URLSearchParams`, then anyway we'd have to update every place where it's in use)

Reply:
ok, i will delete the `require`.